src/modal/modal-ref.ts

```diff
@@ -31,6 +50,23 @@ export class NgbActiveModal {
 	dismiss(reason?: any): void {}
 }
 
+class ModalAttributes {
```

Comment:
why have the whole class for this?

Just `const WINDOW_ATTRIBUTES: string[] = [...]` would be enough

Reply:
As const can not be used for a class member you'll see `readonly` , done